client/my-sites/domains/components/domain-warnings/index.jsx

```diff
@@ -810,26 +806,6 @@ export class DomainWarnings extends PureComponent {
 		);
 	};
 
-	pendingGSuiteTosAcceptanceDomains = () => {
-		const domains = this.getDomains().filter(
-			( domain ) => hasPendingGSuiteUsers( domain ) || isPendingGSuiteTOSAcceptance( domain )
-		);
-
-		if ( domains.length === 0 ) {
-			return null;
-		}
-
-		return (
-			<PendingGSuiteTosNotice
-				isCompact={ this.props.isCompact }
-				key="pending-gsuite-tos-notice"
-				siteSlug={ this.props.selectedSite && this.props.selectedSite.slug }
-				domains={ domains }
-				section="domain-management"
-			/>
```

Comment:
I'm afraid that removing all this code leads to a regression - the `PendingGSuiteTosNotice` component is indeed responsible for rendering the notice in the sidebar when `isCompact` set to `true` (and the idea is to keep it):

![image](https://user-images.githubusercontent.com/594356/140963938-7aa5a87c-53aa-4b23-aa93-f05a698e41ef.png)

I'm not sure if that would be the best approach but maybe we could extract [that code](https://github.com/Automattic/wp-calypso/blob/f2e6f88ae65c8571bbac21b284c29b48ec987598/client/my-sites/domains/components/domain-warnings/pending-gsuite-tos-notice.jsx#L85-L106) here?